draft-ietf-dnsop-svcb-https.md

```diff
@@ -770,8 +770,30 @@ ALPN IDs that do not uniquely identify a protocol suite (e.g. an ID that
 can be used with both TLS and DTLS) are not compatible with this
 SvcParamKey and MUST NOT be included in the ALPN set.
 
+To establish a connection to the endpoint, clients MUST
+
+1. Identify all supported IDs in the ALPN set.
```

Comment:
Nitpick: Not sure I've seen these called "IDs" before. RFC7301 says "protocol" or "protocol name". RFC8446 says "ALPN protocol". RFC7838 says "ALPN protocol name" with a few instances of "ALPN protocol".

Reply:
I've reworked the terminology to be more precise, and use "Identification Sequence", "protocol name", and "protocol" as appropriate.